.github/workflows/test.yaml

```diff
@@ -14,8 +14,6 @@ jobs:
           go-version: ${{ matrix.go-version }}
       - name: Checkout code
         uses: actions/checkout@v2
-      - name: Download test data
-        run: curl -L https://raw.githubusercontent.com/package-url/purl-spec/master/test-suite-data.json -o testdata/test-suite-data.json
```

Comment:
IIRC, this test data download is required, because of a previous issue.

Looks like your PR is not rebased. Any chance you can rebase that on the current main branch? The differences shouldn't be so big.